injected/src/features/broker-protection/actions/click.js

```diff
@@ -63,3 +99,112 @@ function selectRootElement (clickElement, userData, root = document) {
 
     throw new Error('`parent` was present on the element, but the configuration is not supported')
 }
+
+/**
+ * Evaluate a comparator and return the appropriate function
+ * @param {string} operator
+ * @returns {(a: any, b: any) => boolean}
+ */
+export function getComparisonFunction (operator) {
+    switch (operator) {
+    case '=':
```

Comment:
I'm still interested why we need all these three? Same for != and !==. '=' is assignment usually too?

Reply:
I don't think it's clear [in the JSON](https://github.com/duckduckgo/content-scope-scripts/pull/1141/files#diff-56d394cac240644f146ac4a55d38e24230eb34131b8aeb6900c3b9b6a8c569efR10) that these operators will behave according to Javascript rules, so I'm mostly just trying to safeguard against someone less familiar with the internals putting `=` when they almost certainly mean `===` (same goes for `!=` and `!==`).